sdk/security/keyvault/azcertificates/fake/server_test.go

```diff
@@ -0,0 +1,560 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator. DO NOT EDIT.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
```

Comment:
Is this file really generated?

Reply:
fixed in [541e497](https://github.com/Azure/azure-sdk-for-go/pull/24768/commits/541e49720319d36e478f7f287fe62bfe386ca3dd)